### DISCUSSION TITLE
Hoe to proceed for the project after applying.

### DISCUSSION THREAD

**Discussion by hritikbhattacharya** [community/community#49983] [GitHub Education] [answered]
I have applied for a project in octernship , after applying I was assigned a repo by the github classroom , which doesn't contain any thing.
please anyone tell me what I am suppose to do  next .

**Comment by CoffeeGeek101** [comment, marked as answer]
Maybe you should go to the repo of the organisation, you have applied to. They have an Octernships file, it's contains the detailed description on the task and the internship.

For your reference :
https://github.com/appwrite/octernships
This is the `appwrite` octernship folder, it contains all the internship details. Similarly, you can find this kind of files/folder for every organisation in their repo.

I Hope this helps.
